ios/Classes/TiNfcVTagTechnology.m

```diff
@@ -9,4 +9,701 @@
+                                                     @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                   }];
+                                            }];
+}
+
+- (void)resetToReadyWithRequestFlags:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  [[self.tagProxy asNFCISO15693Tag] resetToReadyWithRequestFlags:requestFlags
+                                               completionHandler:^(NSError *_Nullable error) {
+                                                 if (![self _hasListeners:@"didResetToReadyWithRequestFlags"]) {
+                                                   return;
+                                                 }
+                                                 [self fireEvent:@"didResetToReadyWithRequestFlags"
+                                                      withObject:@{
+                                                        @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
+                                                        @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
+                                                        @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                      }];
+                                               }];
+}
+
+- (void)selectWithRequestFlags:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  [[self.tagProxy asNFCISO15693Tag] selectWithRequestFlags:requestFlags
+                                         completionHandler:^(NSError *_Nullable error) {
+                                           if (![self _hasListeners:@"didSelectWithRequestFlags"]) {
+                                             return;
+                                           }
+                                           [self fireEvent:@"didSelectWithRequestFlags"
+                                                withObject:@{
+                                                  @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
+                                                  @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
+                                                  @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                }];
+                                         }];
+}
+
+- (void)stayQuiet:(id)args
+{
+  [[self.tagProxy asNFCISO15693Tag] stayQuietWithCompletionHandler:^(NSError *_Nullable error) {
+    if (![self _hasListeners:@"didStayQuiet"]) {
+      return;
+    }
+    [self _hasListeners:@"didStayQuiet"];
```

Comment:
instead of checking event listener, i think you should fire an event like 
```suggestion
    [self fireEvent:@"didStayQuiet" withObject:@{
                                                  @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
                                                  @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
                                                  @"errorDomain" : error != nil ? [error domain] : [NSNull null]
                                                }];
                                         }];
```

Reply:
Done